app/views/multiyear/transfer/single_year_select.scala.html

```diff
@@ -55,30 +55,28 @@ <h1 class="heading-xlarge" id="heading">@Html(Messages("pages.multi-year-select.
 @helpers.form(action = routes.TransferController.extraYearsAction, args = 'id -> "eligible-years-form") {
 <fieldset class="inline spacingBottom" id="earlierYears">
     <div class="form-group" id="year-@availableYears.head.year">
-        <h2 class="heading-medium"><span style="color: grey">@Html(Messages("pages.multiyear.taxyear", TextGenerators.taxDateIntervalShort(availableYears.head.year, LanguageUtils.isWelsh(messages))))</span>
-            <br>@TextGenerators.taxDateInterval(availableYears.head.year, LanguageUtils.isWelsh(messages))</h2>
+        <h2 class="heading-medium"><span style="color: grey">@Html(Messages("pages.multiyear.taxyear", TextGenerator().taxDateIntervalShort(availableYears.head.year)))</span>
+            <br>@TextGenerator().taxDateInterval(availableYears.head.year)</h2>
```

Comment:
Further suggestion is we dont do this at all here and instead do it further out in a service.. and let the view only do the displaying of data

Reply:
Yes or a viewmodel. The changes here were to reflect the general change to the TextGenerator. Most of the old views take on far too much responsibility and will need fixing on a case by case basis.